moveit_setup_assistant/src/widgets/start_screen_widget.cpp

```diff
@@ -421,6 +421,23 @@ bool StartScreenWidget::loadExistingFiles()
   fs::path kinematics_yaml_path = config_data_->config_pkg_path_;
   kinematics_yaml_path /= "config/kinematics.yaml";
 
+  // Load sensors_3d yaml file if available --------------------------------------------------
+  fs::path sensors_3d_yaml_path = config_data_->config_pkg_path_;
+  sensors_3d_yaml_path /= "config/sensors_3d.yaml";
```

Comment:
add note inside the sensors_3d.yaml files that the filename should not be changed or else the MSA will not detect it